lib/cocoapods/installer.rb

```diff
@@ -206,15 +206,15 @@ def create_generator
     #
     def generate_pods_project(generator = create_generator)
       UI.section 'Generating Pods project' do
-        @target_installation_results = generator.generate!
-        @pods_project = generator.project
+        pod_project_generation_result = generator.generate!
+        @target_installation_results = pod_project_generation_result.target_installation_results
+        @pods_project = pod_project_generation_result.project
```

Comment:
could this not be an ivar?

Reply:
`pod_project_generation_result`? Don't see why it should be if it's not used outside this scope and only passes them off to the existing ivars.

Reply:
I think he means `@pods_project` but for the scope of this PR maybe not change `installer.rb` at all, since it also exposes a public API in `post_install`.